Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -726,7 +726,7 @@ PRINT TITLE="Pref Access SSN Update RPT"
    DEBUGSTATE="UPDATED PA LIST"
    CALL PRINTPALISTTODEBUG
 
-   COL=001 FORMAT("R999999",ACCOUNTCOUNT)
+   COL=001 FORMAT("999999",ACCOUNTCOUNT)
    COL=009 ACCOUNT:NUMBER
    COL=021 SOURCESSN
    COL=041 RIGHT ACCOUNT:TYPE
```

Comment:
Hi Tom, This code looks good overall!

I found one thing, I think.
I think the following procedure, VALIDATEPALIST, could lead to multiple "unrestricted PA records" marking
each other as Invalid.

The reason for this is the first loop goes 1 to PA Count.
Then the Second Loop goes 1 to PA Count.
But if for example if PREFACCESSACCT(1) and PREFACCESSACCT(2) are identical
"unrestricted PA records" then on the first go round PREFACCESSACCT(2) gets
marked as PASTATUSINVALID.
Once you finish the second loop and go back to the first loop then
PREFACCESSACCT(2) gets compared with PREFACCESSACCT(1) out of the gate;
and PREFACCESSACCT(1) also gets marked as PASTATUSINVALID.
To repair this I think you can change
IF PREFACCESSIDTYPE(PALOOP)=2 THEN to
IF PREFACCESSIDTYPE(PALOOP)=2 AND PREFACCESSSTATUS(PALOOP)<>PASTATUSINVALID THEN

Let me know if I'm not getting it, and the logic won't do what I think it would
in that case.

PROCEDURE VALIDATEPALIST
[* The list of Preference access records is cycled through. For each PA record
** which has an ID type of 2 (unrestricted), the list is checked to see if there
** are any other preference access records pointing to the same account but to a
** specific share or loan. If found, the access record is marked for deletion
*]
FOR PALOOP=1 TO PACOUNT
DO
IF PREFACCESSIDTYPE(PALOOP)=2 THEN [Only test PA record if it's unrestricted]
DO
FOR PACHECKLOOP=1 TO PACOUNT
DO
IF PACHECKLOOP<>PALOOP AND [Skip over the PA record currently being tested]
PREFACCESSACCT(PALOOP)=PREFACCESSACCT(PACHECKLOOP) THEN
DO
PREFACCESSSTATUS(PACHECKLOOP)=PASTATUSINVALID
PREFACCESSSTATUSREASON(PACHECKLOOP)="Unrestricted PA record exists"
END
END
END
END
END [PROCEDURE]